server/test/framework/Syntax_Tests.cpp

```diff
@@ -1825,7 +1825,8 @@ namespace {
                          [] (const tests::Tests::MethodTestCase& testCase) {
                              return stoi(testCase.paramValues[0].view->getEntryValue(nullptr)) ==
                                     stoi(testCase.paramValues[1].view->getEntryValue(nullptr)) &&
-                                    testCase.returnValue.view->getEntryValue(nullptr) == "{{{'k', 1.010100e+00}}}";
+                                    StringUtils::startsWith(testCase.returnValue.view->getEntryValue(nullptr),
+                                                 "{from_bytes<StructWithStructInUnion::DeepUnion>({");
```

Comment:
we can get the `from_bytes` init expression, if there is no the single field init strategy. 
That is the case of `in alignment bytes garbage` like a case
```
union A {
    struct {
        char c;
        int hiInt;    
    }
    int lowInt;
}
```
no way to repeat the state in initializer for a state after `a.lowInt = 0x0001; a.hiInt = 0x01;`.